modules/core/04-channel/keeper/events.go

```diff
@@ -173,24 +163,15 @@ func emitRecvPacketEvent(ctx sdk.Context, packet types.Packet, channel types.Cha
 	})
 }
 
-// emitWriteAcknowledgementEvent emits an event that the relayer can query for
+// emitWriteAcknowledgementEvent emits an event for when an acknowledgement is written on the receiving chain.
 func emitWriteAcknowledgementEvent(ctx sdk.Context, packet types.Packet, channel types.Channel, acknowledgement []byte) {
```

Comment:
looks like hermes reconstructs the packet from this event. Will inquire about seeing if this can be changed